app/models/shipit/stack.rb

```diff
@@ -255,6 +254,10 @@ def undeployed_commits
       scope.map.with_index { |c, i| UndeployedCommit.new(c, i) }.reverse
     end
 
+    def last_deploy
```

Comment:
There is already a `last_deployed_commit` method. Basically all you need is to expose it in the serializer.

The rest of the diff shouldn't be required.

Reply:
Oh, nvm, in your case you want the last deploy even if it's not successful.

In this case I think it would be better to add a `/tasks` endpoint and grab the last one.

There only so much stuff that can be put into the StackSerializer.

Reply:
That works for me. What's considered a Task in shipit? just `Deploy`/`Rollback`?

Reply:
`Task > Deploy > Rollback`

Straight `Task` are the custom commands you can trigger, but that don't change the deployed revision. e.g. failover, restart & such.